src/Components/test/testassets/TestServer/ClientStartup.cs

```diff
@@ -0,0 +1,53 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace TestServer
+{
+    // WARNING: DO NOT MODIFY THIS STARTUP CLASS FOR TEST PURPOSES
+    // Most of the client-side tests are executed using the developer host directly, so changing values here won't
+    // affect any client-side test.
```

Comment:
It seems a bit odd to have it then.

Would it be possible to instead invoke the dev server code to host the client-side tests? It should be possible to do that from `Program.cs` in the same way that the tests themselves do.

Reply:
Its fair. I though of doing it that way, but one thing that I like about having a separate startup here is that you can modify it while you are troubleshooting stuff and everything is contained within the same project.

Reply:
Added an option to start from the devhost server.